web/src/client/index_test.ts

```diff
@@ -654,21 +662,25 @@ describe("GET /<installerID>", () => {
 });
 
 describe("GET /<installerID>/join.sh", () => {
+  
   describe("/latest/join.sh", async () => {
     const latestResolve = await Installer.latest().resolve();
-
+    
     it(`injects k8s ${latestResolve.spec.kubernetes.version}, weave ${latestResolve.spec.weave?.version}, rook ${latestResolve.spec.rook?.version}, contour ${latestResolve.spec.contour?.version}, registry ${latestResolve.spec.registry?.version}, prometheus ${latestResolve.spec.prometheus?.version}`, async () => {
       const script = await client.getJoinScript("latest");
 
       expect(script).to.match(new RegExp(`version: ${latestResolve.spec.kubernetes.version}`));
       expect(script).to.match(new RegExp(`version: ${latestResolve.spec.weave?.version}`));
-      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.rook?.version}`));
+      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.minio?.version}`));
+      expect(script).to.match(new RegExp(`version: ${latestResolve.spec.longhorn?.version}`));
```

Comment:
does this list need to include containerd?

Reply:
added it